### DISCUSSION TITLE
(mpfs-pmp-demo) the demo-guide instruction does not work with tag 2021.11 and 2022.02

### DISCUSSION THREAD

**Discussion by okawaj9** [polarfire-soc/polarfire-soc-documentation#24] [Applications and demos] [answered]
1. Issue:
https://github.com/polarfire-soc/polarfire-soc-documentation/blob/master/demo-guides/mpfs-pmp-demo.md
- The PMP demo guide works  with tag 2021.08, only.
- The demo-guide instruction does not work with tag 2021.11 and 2022.02

2. Phenomenon:

2022.02:
HSS boot fails (stopping) with following message
[6.898297] getBootImageFromMMC_(): Preparing to copy from MMC to DDR ...

2021.11:
mpfs-pmp-app-u54-1 application code does not output on UART1.

HSS:
[7.850626] boot_opensbi_init_onExit(): boot_service(u54_1)::u54_1:sbi_init 0x80000000
[7.859740] __assert_func(): services/boot/hss_boot_service.c:698: boot_opensbi_init_onExit() Assertion failed:
        result

UART1 (mpfs-pmp-app-u54-1):
[7.583159] HSS_Boot_PMPSetupHandler(): Hart1 setup complete
sbi_trap_error: hart1: trap handler failed (error -2)
sbi_trap_error: hart1: mcause=0x0000000000000005 mtval=0x000000005000050c
sbi_trap_error: hart1: mepc=0x0000000008017c84 mstatus=0x0000000a00001800
sbi_trap_error: hart1: ra=0x0000000008017c6a sp=0x00000000080416b8
sbi_trap_error: hart1: gp=0x0000000000000000 tp=0x000000000a001000
sbi_trap_error: hart1: s0=0x0000000000000001 s1=0x0000000000000000
sbi_trap_error: hart1: a0=0x0000000000000000 a1=0x0000000008041710
sbi_trap_error: hart1: a2=0x0000000000000001 a3=0x0000000008037628
sbi_trap_error: hart1: a4=0x000000000802b6b0 a5=0x0000000050000500
sbi_trap_error: hart1: a6=0x0000000008036e38 a7=0x0000000008035d30
sbi_trap_error: hart1: s2=0x000000000000000e s3=0x0000000000000002
sbi_trap_error: hart1: s4=0x0000000008004ec0 s5=0x0000000008035910
sbi_trap_error: hart1: s6=0x00000000080217e8 s7=0x000000000801d370
sbi_trap_error: hart1: s8=0x00000000080215b0 s9=0x0000000000000000
sbi_trap_error: hart1: s10=0x0000000000000000 s11=0x0000000000000000
sbi_trap_error: hart1: t0=0x0000000a00001880 t1=0x0000000000000000
sbi_trap_error: hart1: t2=0x0000000000000001 t3=0x0000000000000011
sbi_trap_error: hart1: t4=0x0000000000000000 t5=0x0000000000000000
sbi_trap_error: hart1: t6=0x0000000000000000

**Comment by sc-mchp** [comment, marked as answer]
HSS: 2022.02 problem we are able to reproduce. we are working on it.
